src/explore-education-statistics-admin/src/pages/publication/PublicationEditReleaseSeriesLegacyLinkPage.tsx

```diff
@@ -54,6 +54,7 @@ export default function PublicationEditReleaseSeriesLegacyLinkPage({
     <LoadingSpinner loading={isLoading}>
       <h2>Edit legacy release</h2>
 
+      {/* TODO rename to releaseVersionId */}
```

Comment:
Why not change this one in this PR? Also it would be good to have a ticket number on the TODOs to refer to.